src/test/regress/expected/distributed_collations.out

```diff
@@ -1,8 +1,42 @@
 SET citus.next_shard_id TO 20050000;
+SHOW server_version \gset
+SELECT substring(:'server_version', '\d+')::int > 14 AS server_version_above_fourteen
+\gset
 CREATE USER collationuser;
 CREATE SCHEMA collation_tests AUTHORIZATION collationuser;
 CREATE SCHEMA collation_tests2 AUTHORIZATION collationuser;
 SET search_path to collation_tests;
+\if :server_version_above_fourteen
+-- fail, needs "locale"
```

Comment:
can we have these new tests on pg15.sql such that we avoid alternative output file?